nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -54,6 +55,7 @@ def find_dependency_urls
             end.compact.uniq
         end
 
+        # rubocop:disable Metrics/AbcSize
         def build_url_for_details(repo_details)
```

Comment:
Not for this PR, but I think this method would is a good candidate for some cleanup. Doing a bit more than building a url. Would we want to encapsulate parsing the repo metadata response somewhere else?